From b5d5b79f0b708f3f5d07ce68113e258409767a40 Mon Sep 17 00:00:00 2001
From: Brian Ma <chma0@nuvoton.com>
Date: Thu, 9 May 2024 13:49:20 +0800
Subject: [PATCH 1/2] Update env offset for layout change

And also update uimage_flash_addr due to flash layout change.

Signed-off-by: Brian Ma <chma0@nuvoton.com>
---
 configs/arbel_evb_defconfig | 4 ++--
 include/configs/arbel.h     | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configs/arbel_evb_defconfig b/configs/arbel_evb_defconfig
index a3d0a2142d..eee545aab3 100644
--- a/configs/arbel_evb_defconfig
+++ b/configs/arbel_evb_defconfig
@@ -7,7 +7,7 @@ CONFIG_NR_DRAM_BANKS=2
 CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0x06208000
 CONFIG_ENV_SIZE=0x40000
-CONFIG_ENV_OFFSET=0x3C0000
+CONFIG_ENV_OFFSET=0x7C0000
 CONFIG_ENV_SECT_SIZE=0x1000
 CONFIG_DM_GPIO=y
 CONFIG_DEFAULT_DEVICE_TREE="nuvoton-npcm845-evb"
@@ -16,7 +16,7 @@ CONFIG_DM_RESET=y
 CONFIG_ARCH_NPCM8XX=y
 CONFIG_TARGET_ARBEL_EVB=y
 CONFIG_SYS_LOAD_ADDR=0x06208000
-CONFIG_ENV_ADDR=0x803C0000
+CONFIG_ENV_ADDR=0x807C0000
 CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_USE_BOOTCOMMAND=y
diff --git a/include/configs/arbel.h b/include/configs/arbel.h
index ce94a94476..bee8d21b41 100644
--- a/include/configs/arbel.h
+++ b/include/configs/arbel.h
@@ -24,7 +24,7 @@
 #define SPI3_END_ADDR			0xBFFFFFFF
 
 /* Default environemnt variables */
-#define CFG_EXTRA_ENV_SETTINGS   "uimage_flash_addr=80400000\0"   \
+#define CFG_EXTRA_ENV_SETTINGS   "uimage_flash_addr=80800000\0"   \
 		"stdin=serial\0"   \
 		"stdout=serial\0"   \
 		"stderr=serial\0"    \
-- 
2.34.1

